test/internet/internet.status

```diff
@@ -1,6 +1,7 @@
 prefix internet
 
 test-dns                                : PASS,FLAKY
+test-http-dns-fail                      : PASS,FLAKY
```

Comment:
Possibly related to https://github.com/joyent/node/issues/25858. I ran into this problem also on jenkins.nodejs.org on the Ubuntu 14.04 Jenkins agent, and fixed the DNS server configuration manually.